release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -60,6 +60,63 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-6-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-6-boot-live-iso-by-running-coreos-installer"]
+==== Boot live ISO by running `coreos-installer`
+
+You can now boot an {op-system} live ISO in a live environment by running the
```

Comment:
Discussing the "live" environment should generalize to both ISO and PXE.  ([The enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/liveisoinstall.md) could have worded this better.

How about something more like:

> The PXE media and ISO available for {op-system} are now a fully "live" environment.  Unlike the previous dedicated "installer" PXE media and ISO, the 4.6 live environment can be configured via Ignition and contains all the same packages as the main {op-system} such as `podman`.  This allows arbitrary scripting of pre/post installation workflows (e.g. run `coreos-installer` and then afterwards make an HTTP request to signal success to a provisioning server).
> PXE boots use the normal `ignition.config.url`.  The ISO can be configured via Ignition by using `coreos-installer iso ignition embed`.